code/script.js

```diff
@@ -33,19 +46,124 @@ const showMessage = (message, sender) => {
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000);
+form.addEventListener('submit', (event) => {
+  event.preventDefault()
+  const name = nameInput.value
+  userAnswer(`${name}`)
+  nameInput.value = ''
+  setTimeout(() => dayBooking(), 1000)
+})
+
+// second question dayBooking
+
+const dayBooking = () => { 
+  questionNumber++
+  botAnswer(`Which day would you like to visit us?`);
+
+  inputWrapper.innerHTML = `
+    <button id='fridayButton'>Friday</button>
+    <button id='saturdayButton'>Saturday</button>
+    `
+  document
+    .getElementById('fridayButton')
+    .addEventListener('click', () => nextQuestion('Friday'))
+  document
+    .getElementById('saturdayButton')
+    .addEventListener('click', () => nextQuestion('Saturday'))
+  }
+
+// third question
+
+const timeBooking = (type) => {
+  questionNumber++
+  botAnswer(`Which time on ${type} would you like to book?`)
+  if (type === 'Friday') {
+    inputWrapper.innerHTML=`
+    <select id = 'select'>
+      <option value='' selected disabled> Choose time </option>
+      <option value='Lunch'>Lunch</option>
+      <option value='After Work'>After Work</option>
+    </select> 
+    `
+  } 
+  else {
+    inputWrapper.innerHTML=`
+    <select id = 'select'>
+      <option value='' selected disabled> Choose time </option>
+      <option value='Brunch'>Brunch</option>
+      <option value='Dinner'>Dinner</option>
+    </select> 
+    `
+  }
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion(select.value))
+}
+
+// fourth question 
+
+const guestBooking = (time) => { 
+  questionNumber++
+  botAnswer(`How many guests will you be for ${time}?`);
+
+
+inputWrapper.innerHTML=`
+<select id = 'select'>
+  <option value='' selected disabled> Number of quests </option>
+  <option value='1'>1</option>
+  <option value='2'>2</option>
+  <option value='3'>3</option>
+  <option value='4'>4</option>
+  <option value='5'>5</option>
+</select> 
+`
+
+const select = document.getElementById('select')
+select.addEventListener('change', () => nextQuestion(select.value))
+}
+
+// summary
+
+const finaliseBooking = (choice) => {
+  select.remove();
+  questionNumber++
+
+  if (choice === '1') {
+    showMessage("Oh! We see you brought your best company, Yourself! We'll make sure you get the best table!", 'bot')
+  } else if (choice === '5') {
+    showMessage("We love a big crowd! Please contact us at gm@fakemail.com for large bookings!", 'bot')
+  } else {
+    botAnswer("We look forward to your visit!", 'bot')
+  }
+  
+  select.addEventListener('change', () => nextQuestion(inputValue))
```

Comment:
we get an error message on this select.addEventListener. We think you are trying to use the same constant used on line 148. Perhaps you can use let instead or create a new variable for this AddEventListener.